File: 动画/eWatch/eWatch/ViewController.m
//
//  ViewController.m
//  eWatch
//
//  Created by Abraham on 9/13/15.
//  Copyright © 2015 Abraham. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    i = 9999;
    timer = [NSTimer scheduledTimerWithTimeInterval:1 target:self selector:@selector(show) userInfo:nil repeats:YES];
    vl.length = 4;
    [vl showText:1];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
-(void)show{
    i--;
    printf("1");
    [vl showText:vl.number +1];
    if(i==0){
        [timer invalidate];
    }
}
@end

File: db/db/ViewController.swift
//
//  ViewController.swift
//  db
//
//  Created by Abraham on 11/8/15.
//  Copyright © 2015 Abraham. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var db:SQLiteDB!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        db = SQLiteDB.sharedInstance()

        db.execute("create table if not exists t_user(uid integer primary key,uname varchar(20),mobile varchar(20))")
        db.execute("insert into t_user (uname,mobile) values('1','18017304908')")
        // Do any additional setup after loading the view, typically from a nib.
        
        initUser()
    }
    
    //从SQLite加载数据
    func initUser() {
        let data = db.query("select * from t_user")
        if data.count > 0 {
            //获取最后一行数据显示
            let user = data[data.count - 1] as SQLRow
            print(user["uname"]!.asString() + user["mobile"]!.asString())
        }
    }

    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

